src/applications/beta-enrollment/routes.js

```diff
@@ -10,7 +10,7 @@ export const features = {
 const routes = {
   path: '/',
   childRoutes: [
-    { path: 'personalization', component: createBetaEnrollmentButton(features.dashboard, '/dashboard-beta') }
+    { path: 'personalization', component: createBetaEnrollmentButton(features.dashboard, '/dashboard') }
```

Comment:
I left this just as an example, but since no content is registered to it it won't have an effect.

Reply:
@ncksllvn do we want to clean up the beta-enrollment/personalization stuff that is hanging around? I think the content files (`content/beta-enrollment`) and the manifest allow that to be accessible.

Reply:
I deleted `content/pages/beta-enrollment/personalization.md` but we can make modifications here too if you'd prefer.